src/gpu/opengl/opengl_texture.cpp

```diff
@@ -729,12 +729,14 @@ bool GLDriver::checkActiveSamplers()
       auto depth_compare_function = getTextureCompareFunction(sq_tex_sampler_word0.DEPTH_COMPARE_FUNCTION());
       gl::glSamplerParameteri(sampler.object, gl::GL_TEXTURE_COMPARE_FUNC, static_cast<gl::GLint>(depth_compare_function));
 
-      // Setup texture LOD
+      // Setup texture LOD/BIAS
       auto min_lod = sq_tex_sampler_word1.MIN_LOD();
       auto max_lod = sq_tex_sampler_word1.MAX_LOD();
       auto lod_bias = sq_tex_sampler_word1.LOD_BIAS();
 
-      // TODO: GL_TEXTURE_MIN_LOD, GL_TEXTURE_MAX_LOD, GL_TEXTURE_LOD_BIAS
+      gl::glSamplerParameteri(sampler.object, gl::GL_TEXTURE_MIN_LOD, static_cast<gl::GLint>(min_lod));
```

Comment:
Are these supposed to be integers?
The OpenGL documentation for GL_TEXTURE_MIN_LOD for example says
```
 Sets the minimum level-of-detail parameter. This floating-point value limits the selection of highest resolution mipmap (lowest mipmap level). The initial value is -1000. 
```